libai/__init__.py

```diff
@@ -262,6 +265,55 @@ def _load_chat(chat: str) -> str:
         return f.read()
 
 
+def save_sqlite(message, completion, system_message, path, query_time, response_time):
+    db_path = Path(CFG.saved_chats_dir) / "chats.db"
+    conn = sqlite3.connect(str(db_path))
+    conn.execute(
+        """CREATE TABLE IF NOT EXISTS chats
+(id INTEGER PRIMARY KEY, text TEXT NOT NULL, role TEXT NOT NULL, model TEXT NOT NULL, created_at DATETIME NOT NULL, token_count INTEGER NOT NULL, md_path TEXT DEFAULT NULL);"""
+    )
+    c = conn.cursor()
+    insert_q = "INSERT INTO chats (text, role, model, created_at, token_count, md_path) VALUES (?,?,?,?,?,?)"
+    c.execute(
+        insert_q,
+        (
+            message["content"],
+            message["role"],
+            completion["model"],
+            query_time,
+            completion["usage"]["prompt_tokens"],
+            path,
+        ),
+    )
+    for choice in completion["choices"]:
+        print(choice)
```

Comment:
Can You make this a `_dbg` ?

Reply:
ah this was just a random bug fixing print, unintentional